heroclass_tina.rb

```diff
@@ -0,0 +1,109 @@
+class Superhero
+  
+   $heroes = [{name: "Batman", identity: "Bruce Wayne", weakness: "Woman"},
+  {name: "Spiderman", identity: "Peter Parker", weakness: "Mary Jane"},
+  {name: "Superman", identity: "Clark Kent", weakness: "Louis Lane"}]
+    input = ''
+  @@herohash = {}
+  def Superhero.get_users
```

Comment:
This is probably not the right case to be using a class method and class variable. But A for trying to use what you learned!  You could actually just pass the list in the object.